File: C/Learn/8BIGGER-OF-TWO-NUMBERS/src/BIGGER-OF-TWO-NUMBERS.c
/*
 ============================================================================
 Name        : BIGGER-OF-TWO-NUMBERS.c
 Author      : 
 Version     :
 Copyright   : Your copyright notice
 Description : Hello World in C, Ansi-style
 ============================================================================
 */

#include <stdio.h>
#include <stdlib.h>

int main(void) {
	int num1, num2;
	setbuf(stdout, NULL);
	printf("Enter the two numbers to be compared : ");
	scanf("%d%d", &num1, &num2);
	if (num1>num2) {
		printf("%d is greater than %d", num1, num2);

	}
	else if (num2>num1) {
		printf("%d is greater than %d", num2, num1);
	}
	else {
		printf("Both are equal");

	}
	return EXIT_SUCCESS;
}

File: JAVA/Learning_0/JAVA-100K/BiggerOfTwoNumbers.java
import java.util.Scanner;

public class BiggerOfTwoNumbers {
    public static void main(String ar[]) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter two numbers:");

        int num1 = sc.nextInt();
        int num2 = sc.nextInt();
        if (num1>num2) {
            System.out.println(num1+" is greatest");

        } else if (num2>num1) {
            System.out.println(num2 + " is greatest");
        } else {
            System.out.println("Both are equal");
        }

        sc.close();
    }
}
